ts/src/bitmex.ts

```diff
@@ -1726,38 +1722,35 @@ export default class bitmex extends Exchange {
         } else {
             filled = cumQty;
         }
-        const id = this.safeString (order, 'orderID');
-        const type = this.safeStringLower (order, 'ordType');
-        const side = this.safeStringLower (order, 'side');
-        const clientOrderId = this.safeString (order, 'clOrdID');
-        const timeInForce = this.parseTimeInForce (this.safeString (order, 'timeInForce'));
-        const stopPrice = this.safeNumber (order, 'stopPx');
         const execInst = this.safeString (order, 'execInst');
         let postOnly = undefined;
         if (execInst !== undefined) {
             postOnly = (execInst === 'ParticipateDoNotInitiate');
         }
+        const timestamp = this.parse8601 (this.safeString (order, 'timestamp'));
```

Comment:
@Dan-krm something is wrong here, check for instance cost, it should be 100 but it's 3kk

```
p bitmex fetchClosedOrders None None 1 --sandbox
Python v3.11.5
CCXT v4.1.32
bitmex.fetchClosedOrders(None,None,1)
[{'amount': 100.0,
  'average': 37522.0,
  'clientOrderId': None,
  'cost': 3752200.0,
  'datetime': '2022-02-04T14:14:46.473Z',
  'fee': None,
  'fees': [],
  'filled': 100.0,
  'id': '4a69755b-3e0a-4d05-812b-8e653b0968d9',
  'info': {'account': '395724',
           'avgPx': '37522',
           'clOrdID': '',
           'clOrdLinkID': '',
           'contingencyType': '',
           'cumQty': '100',
           'currency': 'USD',
           'displayQty': None,
           'execInst': '',
           'leavesQty': '0',
           'ordRejReason': '',
           'ordStatus': 'Filled',
           'ordType': 'Market',
           'orderID': '4a69755b-3e0a-4d05-812b-8e653b0968d9',
           'orderQty': '100',
           'pegOffsetValue': None,
           'pegPriceType': '',
           'price': '37522',
           'settlCurrency': 'XBt',
           'side': 'Sell',
           'stopPx': None,
           'symbol': 'XBTUSD',
           'text': 'Submission from testnet.bitmex.com',
           'timeInForce': 'ImmediateOrCancel',
           'timestamp': '2022-02-04T14:14:46.473Z',
           'transactTime': '2022-02-04T14:14:46.473Z',
           'triggered': '',
           'workingIndicator': False},
  'lastTradeTimestamp': 1643984086473,
  'lastUpdateTimestamp': None,
  'postOnly': False,
  'price': 37522.0,
  'reduceOnly': None,
  'remaining': 0.0,
  'side': 'sell',
  'status': 'closed',
  'stopLossPrice': None,
  'stopPrice': None,
  'symbol': 'BTC/USD:BTC',
  'takeProfitPrice': None,
  'timeInForce': 'IOC',
  'timestamp': 1643984086473,
  'trades': [],
  'triggerPrice': None,
  'type': 'market'}]
```